spec/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -153,3 +153,95 @@
     end.to output("Couldn't convert 1 chapter(s): #{dewplayer_work.first_chapter.id}\nConverted 0 chapter(s).\n").to_stdout
   end
 end
+
+describe "rake After:fix_teen_and_up_imported_rating" do
+  let(:noncanonical_teen_rating) { Rating.create(name: "Teen & Up Audiences", canonical: false) }
+  let(:canonical_gen_rating) { Rating.find_or_create_by(name: ArchiveConfig.RATING_GENERAL_TAG_NAME, canonical: true) }
+  let!(:canonical_teen_rating) { Rating.find_or_create_by(name: ArchiveConfig.RATING_TEEN_TAG_NAME, canonical: true) }
+  let(:work_with_noncanonical_rating) { create(:work) }
```

Comment:
You can create a work with tags in one go, like this:

https://github.com/otwcode/otwarchive/blob/9fa6c098237352c2386c6eab7d911d63c5d263c5/spec/lib/tasks/tag_tasks.rake_spec.rb#L18